recirq/benchmarks/rabi_oscillations.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Iterator, List, Optional, Sequence, Tuple, TYPE_CHECKING
+import numpy as np
+import sympy
+
+from matplotlib import pyplot as plt
+
+# this is for older systems with matplotlib <3.2 otherwise 3d projections fail
+from mpl_toolkits import mplot3d  # pylint: disable=unused-import
+from cirq import circuits, ops, study
+
+if TYPE_CHECKING:
+    import cirq
+
+
+class RabiResult:
+    """Results from a Rabi oscillation experiment."""
```

Comment:
This docstring doesn't really explain more than the name of the class.  Consider adding more information.